server/lib/packets/packet.go

```diff
@@ -92,16 +103,39 @@ func (pkt *RecvPacket) Process(t Transport) {
 	}
 }
 
+func Actor2ProtoError(aerr *yue.ActorError) *proto.Err {
+	if aerr.Is(yue.ActorUnavailable) {
+		return &proto.Err {
+			Type: proto.Error_TemporaryUnavailable,
+		}
+	} else {
```

Comment:
__[gometalinter]__
* [golint] if block ends with a return statement, so drop this else and outdent its block